packages/apptive_grid_form/pubspec.yaml

```diff
@@ -16,16 +16,16 @@ dependencies:
     sdk: flutter
   flutter_svg: ^2.0.5
   flutter_typeahead: ^4.3.8
-  geolocator: ^9.0.2
+  geolocator: ^10.0.1
   google_maps_flutter: ^2.2.6
   google_maps_flutter_platform_interface: ^2.2.6
   hand_signature: ^3.0.1
   http: ^0.13.6
```

Comment:
As discussed the http dependencies should be defined to allow `>=0.13.6 <2.0.0`

Reply:
done